### PR TITLE
Emergency bugfix for experimental 4938

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Fixed the horrible mess I made.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
Would @mugling @Rivet-the-Zombie or somebody please merge this?

**Comment by BorkBorkGoesTheCode** [comment]
Are you still online @Cyrano7 ?

**Comment by Cyrano7** [comment]
I don't have merge rights.

File: data/json/items/gun/9mm.json [hunk 1]
```diff
@@ -29,6 +29,13 @@
       [ "BURST", "3 rd.", 3 ],
       [ "AUTO", "auto", 5 ]
     ],
+    "magazines" : [
```
Comment (Cyrano7) [inline data/json/items/gun/9mm.json]: Although you don't need this space between " and :

**Comment by Podesta** [comment]
Make a post over at reddit/forums telling people not to update to it. It'll probably get merged in a couple hours.

File: data/json/items/gun/9mm.json [hunk 2]
```diff
@@ -29,6 +29,13 @@
       [ "BURST", "3 rd.", 3 ],
       [ "AUTO", "auto", 5 ]
     ],
+    "magazines": [
```
Comment (kevingranade) [inline data/json/items/gun/9mm.json]: How is this a bugfix?

File: data/json/items/gun/metal_rail.json [hunk 3]
```diff
@@ -1,32 +1,5 @@
 [
   {
-    "id": "rebar_rifle",
```
Comment (kevingranade) [inline data/json/items/gun/metal_rail.json]: why?

File: data/json/items/gun/9mm.json (hunk 2 above)
Comment (BorkBorkGoesTheCode) [inline data/json/items/gun/9mm.json]: My PR removed the magazines by accident

File: data/json/items/gun/metal_rail.json (hunk 3 above)
Comment (BorkBorkGoesTheCode) [inline data/json/items/gun/metal_rail.json]: For some reason git rebase put that there

File: data/json/items/gun/9mm.json (hunk 2 above)
Comment (BorkBorkGoesTheCode) [inline data/json/items/gun/9mm.json]: Starting a world in experimental 4938 produces errors related to ammo/222.json, ammo/545x39.json, ammo/9x18.json, ammo/metal_rail.json, gun/223.json, gun/308.json, and gun/9mm.json.

**Comment by BorkBorkGoesTheCode** [comment]
@Coolthulhu  are you online now?

**Comment by Coolthulhu** [comment]
Yes, I'm trying to find out what's the deal with all the broken JSONs.

**Comment by Coolthulhu** [comment]
Still warns about missing magazine for one gun

**Comment by Coolthulhu** [comment]
The bugged item is 545_ap - it is missing a symbol and has wrong ammo type.

**Comment by BorkBorkGoesTheCode** [comment]
That is really weird. The copy from process worked before

**Comment by Coolthulhu** [comment]
I added a fix on my side. For now just copied the relevant values manually.
Master branch should work now.

**Comment by BorkBorkGoesTheCode** [comment]
Closes #16907

**Comment by BorkBorkGoesTheCode** [comment]
Thank you.

**Comment by kevingranade** [comment]
I'd like answers to my questions about parts of this change.  Several pieces looked like non-bugfixes, if you're making an "emergency bugfix" you should never include additional changes.

**Comment by BorkBorkGoesTheCode** [comment]
For some reason git rebase decided to do strange things, probably because I used the wrong setting. I do not have enough experience with git to tell you what it was doing.

**Comment by BorkBorkGoesTheCode** [comment]
From now on I'm going to avoid rebase until I know the code is ready.

**Comment by BorkBorkGoesTheCode** [comment]
The rebar_rail ordering was part of the original PR. it can be found at the end of the gun update PR #16863 .

**Comment by BorkBorkGoesTheCode** [comment]
@kevingranade are you still online?

**Comment by kevingranade** [comment]
More than avoiding rebase, get in the habit of inspecting your changes
every time you adjust them.
All kinds of merge/rebase operations can do unexpected things, the only way
to prevent problems is to review the changes.

**Pull request merged**